samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -129,20 +129,22 @@ Configuring models
 
 See :ref:`ug_bt_mesh_model_config_app` for details on how to configure the mesh models with the nRF Mesh mobile app.
 
-Configure the Generic OnOff Client model on each element on the :guilabel:`Mesh Light Fixture` node:
+Configure the Generic OnOff Server model on each element on the :guilabel:`Mesh Light Fixture` node:
 
 * Bind the model to :guilabel:`Application Key 1`.
 
   Once the model is bound to the application key, you can control the first LED on the device.
-* In the model view, tap :guilabel:`ON` (one of the Generic On Off Controls) to light up the first LED on the development kit.
+* In the model view, tap :guilabel:`ON` on the second Generic On Off Server to light up the first LED on the development kit.
```

Comment:
```suggestion
* In the model view of the Generic OnOff Server in the second element, tap :guilabel:`ON`.
```

The resulting sequence is described below.

Reply:
I still kind of find it a bit hard to read.

I haven't tested this myself, so I am not familiar with the UI you see when you are actually doing this. Do you select the element first? Or do you go to the model view, see the model (OnOff Server) and select what element? Is it possible to send me a screen shot, so I know what we are describing?

Reply:
You are correct. This can be made more clear. I'll come up with a suggestion.

Reply:
Thanks for the screen shots and the nice discussion, I like the solution.

I wonder if Trond meant that "You should now see the following actions:" should be replaced with "The resulting sequence is described below."? It's the only thing left then.